models/staging/stg_metering_history.sql

```diff
@@ -1,17 +1,22 @@
-{{ config(materialized='incremental') }}
+{{ config(
+    materialized='incremental', 
+    unique_key=['service_type', 'start_time', 'entity_id'],
+) }}
```

Comment:
@NiallRees `entity_id` is a new field in this model.

Because it is part of the `unique_key`, I believe everyone will have to full refresh in order for this to work.

Should we do a major version change as a result?

Reply:
If we wanted to avoid adding `entity_id` to this, what could we do instead? Is this just for
> Use metering_history instead of warehouse_metering_history for consistency

IMO it's fine to use either and we can expect them to be consistent, if it means we can avoid doing comms around a full refresh

Reply:
@NiallRees the data changes here slightly so we need the lookback window in in both `stg_metering_history` and `warehouse_metering_history` to re-process any data that may have changed.

`entity_id` is required for de-deduping, there's no way around it.

Reply:
Ah damn ok. Yeah major version change